uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Hook/ValidateForm/AdminFormOptionsValidation.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Class CRM_HRCore_Hook_ValidateForm_AdminFormOptionsValidation
+ */
+class CRM_HRCore_Hook_ValidateForm_AdminFormOptionsValidation {
+
+  /**
+   * Determines what happens if the hook is handled. Basically the hook
+   * is only handled for option values of the 'toil_amounts' and 'leave_days_amounts'
+   * option group. It basically ensures that the value of the value field of the
+   * option values for these option groups are validated before submission and
```

Comment:
> ensures that the value of the value field of the option values for these option groups are validated

This makes sense after re-reading it a few times, but it hurts my head :crying_cat_face: 

Not asking you to change it though, just lamenting the language required here :smile: